execute/aggregate.go

```diff
@@ -22,7 +21,7 @@ type aggregateTransformation struct {
 
 type AggregateConfig struct {
 	plan.DefaultCost
-	Columns []string `json:"columns"`
+	Columns []string `json:"column"`
```

Comment:
Why is this a slice?

Reply:
I had an idea to use this code to implement a multi-aggregate result (e.g. to computer count + sum in one pass).  so I left it as a slice for now to make that work easier to complete.

Reply:
👍